### ISSUE TITLE
Moving a user Without a Room raises AttributeError

### ISSUE THREAD

**Issue by lukasjuhrich** [agdsn/pycroft#120]
When moving a User who currently has `user.room is None` in the UI, an `AttributeError` is raised.

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.4/dist-packages/werkzeug/contrib/profiler.py", line 103, in __call__
    p.runcall(runapp)
  File "/usr/lib/python3.4/cProfile.py", line 109, in runcall
    return func(*args, **kw)
  File "/usr/local/lib/python3.4/dist-packages/werkzeug/contrib/profiler.py", line 96, in runapp
    appiter = self._app(environ, catching_start_response)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.4/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.4/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.4/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/pycroft/web/blueprints/user/__init__.py", line 431, in move
    form.building.data = user.room.building
AttributeError: 'NoneType' object has no attribute 'building'
```

**Comment by sebschrader** [comment]
Thank you for the fixes, @ibot3.

In the future, please follow the typical [git conventions](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project). In particular, put instructions for GitHub into the body of the commit message and use _imperative present_ instead of _past tense_. E.g. _Fix_ instead of _Fixed_. 

A better commit message would have been for example:

> Handle user without room in move form
>
> Closes #120

instead of

> Fixed moving users when no current room set, closes #120

**Comment by ibot3** [comment]
@sebschrader 
My newest commit unfortunately also do not follow the conventions you mentioned, because I did not see this notice.
I will try to avoid this in the future.
